app/services/create_user.rb

```diff
@@ -0,0 +1,33 @@
+require "./lib/auth0_api"
+
+class CreateUser
```

Comment:
This service is refactored in a later commit to split the responsibilities of creating local versus creating in auth0.